convert2rhel/actions/pre_ponr_changes/handle_packages.py

```diff
@@ -146,14 +146,21 @@ def run(self):
         # shows which packages were not removed, if false, all packages were removed
         pkgs_not_removed = sorted(frozenset(pkghandler.get_pkg_nevras(all_pkgs)).difference(pkgs_removed))
         if pkgs_not_removed:
-            message = "The following packages were not removed: %s" % ", ".join(pkgs_not_removed)
+            message = "The following packages cannot be removed: %s" % ", ".join(pkgs_not_removed)
             logger.warning(message)
             self.add_message(
                 level="WARNING",
                 id="SPECIAL_PACKAGES_NOT_REMOVED",
-                title="Special packages not removed",
-                description="Special packages which could not be removed",
-                diagnosis=message,
+                title="Some packages cannot be removed",
```

Comment:
Maybe we should also update the id? It feels strange that in the id we call them as "special" and in the title is just "some packages"